security/pkg/nodeagent/sds/server.go

```diff
@@ -204,12 +204,12 @@ func (s *sdsservice) debugHTTPHandler(w http.ResponseWriter, req *http.Request)
 		w.WriteHeader(http.StatusInternalServerError)
 		failureMessage := fmt.Sprintf("debug endpoint failure: %s", err)
 		if _, err := w.Write([]byte(failureMessage)); err != nil {
-			log.Errorf("debug endpoint failed to write error response: %s", err)
+			sdsServiceLog.Errorf("debug endpoint failed to write error response: %s", err)
 		}
 		return
 	}
 	if _, err := w.Write([]byte(workloadJSON)); err != nil {
-		log.Errorf("debug endpoint failed to write response: %s", err)
+		sdsServiceLog.Errorf("debug endpoint failed to write response: %s", err)
```

Comment:
there is probably another one still in this file or you would not need to import `log`?

Reply:
Thanks for the catch, there was indeed another `log` call left.